one.alynx.FlipClock.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: one.alynx.FlipClock
 runtime: org.freedesktop.Platform
-runtime-version: "21.08"
+runtime-version: "23.08"
```

Comment:
why not 24.08 ?

Reply:
Also 590 apps use 23.08, 287 apps use 24.08.